tests/integration/workflows/test_workflows_manual_merge.py

```diff
@@ -73,7 +71,12 @@ def test_manual_merge_existing_records(workflow_app):
         current_user_id=1,
     )
 
-    do_resolve_manual_merge_wf(workflow_app, obj_id)
+    # assuming MergeApproval.resolve() is called by the api
```

Comment:
and this change? you adapted the function in https://github.com/inspirehep/inspire-next/pull/3129/files#diff-45d44c66955f78a020f91bbab856b800R65

Reply:
leftover of a previous refactor T.T